code/modules/vehicles/mecha/mecha_defense.dm

```diff
@@ -71,28 +71,43 @@
 	return attack_hand(user, modifiers)
 
 /obj/vehicle/sealed/mecha/attack_alien(mob/living/user, list/modifiers)
+	if(!user.combat_mode)
+		var/datum/element/mech_prying/prying = SSdcs.GetElement(list(/datum/element/mech_prying, user))
```

Comment:
Wat? Noooooo

 isn't there an attack alien signal you can hook? This is not good

Reply:
i think it's fine actually you just lack the knowledge to understand it